### DISCUSSION TITLE
Change Default Map

### DISCUSSION THREAD

**Discussion by ELPatitoLoco** [TAServers/issues#9] [Ideas]
Recently the server crashes alot, and we always get back to gm_construct, it would be good if server could auto change map to the previous one or change the default map with **gm_construct_in_flatgrass**(because atleast most people like that map)

**Comment by Kilometres** [comment]
I agree, the server should default to the previous map (if possible) or rather `gm_bigcity_improved` over `gm_construct_in_flatgrass` as big city seems to have better player retention.

**Reply by Derpius** [reply]
The issue with persisting the map across restarts is that the default map is defined in the server config file, meaning we can't swap it out programmatically from Lua alone (except if we changed map immediately after starting, but that will effectively double the server start time).

For now the simple solution is to just change the hardcoded map over, I'll create a poll

**Comment by Derpius** [comment]
Poll opened here: https://github.com/orgs/TAServers/discussions/15
